hc/front/views.py

```diff
@@ -552,3 +552,36 @@ def privacy(request):
 
 def terms(request):
     return render(request, "front/terms.html", {})
+
+
+# Failed Jobs
+@login_required
+def failed_jobs(request):
+    q = Check.objects.filter(user=request.team.user).order_by("created")
+    checks = list(q)
+
+    counter = Counter()
+    down_tags = set()
+    down_checks = []
+    for check in checks:
+        status = check.get_status()
+        for tag in check.tags_list():
```

Comment:
If we are able to filter with the status as per the preceding review comment, we can do alway with this nested loop.
We will just retrieve non blank tags from the check tag_lists since all the checks retrieved are _downed_ checks. 

Ex:
```
all_down_tags = [ check.tags_list() for check in checks ]
all_nonblank_down_tags = [tag for tag in all_down_tags if tag != '']
counted_down_tags = Counter(all_nonblank_down_tags) # this is a better way to use counter
```